src/streams/rtx_cache.rs

```diff
@@ -48,7 +51,7 @@ impl RtxCache {
     }
 
     pub fn get_cached_packet_smaller_than(&mut self, max_size: usize) -> Option<&mut RtpPacket> {
-        let quantized_size = max_size / RTX_CACHE_SIZE_QUANTIZER;
+        let quantized_size = (max_size / RTX_CACHE_SIZE_QUANTIZER).min(RTX_CACHE_MAX_PACKET_SIZE);
```

Comment:
This is comparing apples and pears. The `quantized_size` is an index, and `RTX_CACHE_MAX_PACKET_SIZE` is a byte size.